From ed044ae2c6b65ee1e1b79402aaaef93f84d1c147 Mon Sep 17 00:00:00 2001
From: napari-bot <81196843+napari-bot@users.noreply.github.com>
Date: Wed, 9 Aug 2023 02:55:46 +0200
Subject: [PATCH] [Automatic] Constraints upgrades: `dask`, `hypothesis`,
 `jsonschema`, `numpy`, `pygments`, `rich`, `superqt` (#6124)

This PR is automatically created and updated by this napari GitHub
Action:

https://github.com/napari/napari/tree/main/.github/workflows/upgrade_test_constraints.yml
It ensures that dependencies are tested with the most recent version.

The updated packages are:

 * cmake
 * comm
 * dask
 * fonttools
 * hypothesis
 * jsonschema
 * numpy
 * pygments
 * referencing
 * rich
 * superqt
 * wheel

---------

Co-authored-by: alisterburt <alisterburt@users.noreply.github.com>
Co-authored-by: Grzegorz Bokota <bokota+github@gmail.com>
---
 napari/_tests/test_conftest_fixtures.py       |  2 +-
 resources/constraints/constraints_py3.10.txt  | 24 ++++++++---------
 resources/constraints/constraints_py3.11.txt  | 24 ++++++++---------
 resources/constraints/constraints_py3.8.txt   | 22 ++++++++--------
 resources/constraints/constraints_py3.9.txt   | 26 +++++++++----------
 .../constraints_py3.9_examples.txt            | 24 ++++++++---------
 resources/requirements_mypy.txt               |  8 +++---
 7 files changed, 65 insertions(+), 65 deletions(-)

diff --git a/napari/_tests/test_conftest_fixtures.py b/napari/_tests/test_conftest_fixtures.py
index de327523..bf959fec 100644
--- a/napari/_tests/test_conftest_fixtures.py
+++ b/napari/_tests/test_conftest_fixtures.py
@@ -65,7 +65,7 @@ def test_disable_throttle(start_mock):

 @patch("qtpy.QtCore.QTimer.start")
 @patch("qtpy.QtCore.QTimer.isActive", return_value=True)
-def test_lack_disable_throttle(start_mock, _active_mock, monkeypatch):
+def test_lack_disable_throttle(_active_mock, start_mock, monkeypatch):
     """This is test showing that if we do not use disable_throttling then timer is started"""
     mock = Mock()

diff --git a/resources/constraints/constraints_py3.10.txt b/resources/constraints/constraints_py3.10.txt
index b5e407e6..b034be78 100644
--- a/resources/constraints/constraints_py3.10.txt
+++ b/resources/constraints/constraints_py3.10.txt
@@ -43,9 +43,9 @@ click==8.1.6
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.27.0
+cmake==3.27.1
     # via triton
-comm==0.1.3
+comm==0.1.4
     # via ipykernel
 contourpy==1.1.0
     # via matplotlib
@@ -53,7 +53,7 @@ coverage==7.2.7
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.7.1
+dask==2023.8.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -80,7 +80,7 @@ filelock==3.12.2
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.41.1
+fonttools==4.42.0
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -92,7 +92,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.82.0
+hypothesis==6.82.2
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -127,7 +127,7 @@ jinja2==3.1.2
     #   numpydoc
     #   sphinx
     #   torch
-jsonschema==4.18.4
+jsonschema==4.19.0
     # via napari (setup.cfg)
 jsonschema-specifications==2023.7.1
     # via jsonschema
@@ -195,7 +195,7 @@ npe2==0.7.2
     #   napari-plugin-manager
 numcodecs==0.11.0
     # via zarr
-numpy==1.25.1
+numpy==1.25.2
     # via
     #   contourpy
     #   dask
@@ -307,7 +307,7 @@ pydantic==1.10.12
     #   app-model
     #   napari (setup.cfg)
     #   npe2
-pygments==2.15.1
+pygments==2.16.1
     # via
     #   ipython
     #   napari (setup.cfg)
@@ -400,7 +400,7 @@ qtpy==2.3.1
     #   napari-plugin-manager
     #   qtconsole
     #   superqt
-referencing==0.30.0
+referencing==0.30.2
     # via
     #   jsonschema
     #   jsonschema-specifications
@@ -408,7 +408,7 @@ requests==2.31.0
     # via
     #   pooch
     #   sphinx
-rich==13.5.0
+rich==13.5.2
     # via
     #   napari (setup.cfg)
     #   npe2
@@ -456,7 +456,7 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 stack-data==0.6.2
     # via ipython
-superqt==0.4.1
+superqt==0.5.0
     # via
     #   magicgui
     #   napari (setup.cfg)
@@ -531,7 +531,7 @@ vispy==0.13.0
     #   napari-svg
 wcwidth==0.2.6
     # via prompt-toolkit
-wheel==0.41.0
+wheel==0.41.1
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/constraints/constraints_py3.11.txt b/resources/constraints/constraints_py3.11.txt
index 7751a4ea..4bf09eab 100644
--- a/resources/constraints/constraints_py3.11.txt
+++ b/resources/constraints/constraints_py3.11.txt
@@ -43,9 +43,9 @@ click==8.1.6
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.27.0
+cmake==3.27.1
     # via triton
-comm==0.1.3
+comm==0.1.4
     # via ipykernel
 contourpy==1.1.0
     # via matplotlib
@@ -53,7 +53,7 @@ coverage==7.2.7
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.7.1
+dask==2023.8.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -76,7 +76,7 @@ filelock==3.12.2
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.41.1
+fonttools==4.42.0
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -88,7 +88,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.82.0
+hypothesis==6.82.2
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -123,7 +123,7 @@ jinja2==3.1.2
     #   numpydoc
     #   sphinx
     #   torch
-jsonschema==4.18.4
+jsonschema==4.19.0
     # via napari (setup.cfg)
 jsonschema-specifications==2023.7.1
     # via jsonschema
@@ -191,7 +191,7 @@ npe2==0.7.2
     #   napari-plugin-manager
 numcodecs==0.11.0
     # via zarr
-numpy==1.25.1
+numpy==1.25.2
     # via
     #   contourpy
     #   dask
@@ -303,7 +303,7 @@ pydantic==1.10.12
     #   app-model
     #   napari (setup.cfg)
     #   npe2
-pygments==2.15.1
+pygments==2.16.1
     # via
     #   ipython
     #   napari (setup.cfg)
@@ -394,7 +394,7 @@ qtpy==2.3.1
     #   napari-plugin-manager
     #   qtconsole
     #   superqt
-referencing==0.30.0
+referencing==0.30.2
     # via
     #   jsonschema
     #   jsonschema-specifications
@@ -402,7 +402,7 @@ requests==2.31.0
     # via
     #   pooch
     #   sphinx
-rich==13.5.0
+rich==13.5.2
     # via
     #   napari (setup.cfg)
     #   npe2
@@ -450,7 +450,7 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 stack-data==0.6.2
     # via ipython
-superqt==0.4.1
+superqt==0.5.0
     # via
     #   magicgui
     #   napari (setup.cfg)
@@ -518,7 +518,7 @@ vispy==0.13.0
     #   napari-svg
 wcwidth==0.2.6
     # via prompt-toolkit
-wheel==0.41.0
+wheel==0.41.1
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/constraints/constraints_py3.8.txt b/resources/constraints/constraints_py3.8.txt
index 26927819..99a6f863 100644
--- a/resources/constraints/constraints_py3.8.txt
+++ b/resources/constraints/constraints_py3.8.txt
@@ -43,9 +43,9 @@ click==8.1.6
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.27.0
+cmake==3.27.1
     # via triton
-comm==0.1.3
+comm==0.1.4
     # via ipykernel
 contourpy==1.1.0
     # via matplotlib
@@ -80,7 +80,7 @@ filelock==3.12.2
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.41.1
+fonttools==4.42.0
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -92,7 +92,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.82.0
+hypothesis==6.82.2
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -108,7 +108,7 @@ importlib-metadata==6.8.0
     #   dask
     #   jupyter-client
     #   sphinx
-importlib-resources==6.0.0
+importlib-resources==6.0.1
     # via
     #   jsonschema
     #   jsonschema-specifications
@@ -135,7 +135,7 @@ jinja2==3.1.2
     #   numpydoc
     #   sphinx
     #   torch
-jsonschema==4.18.4
+jsonschema==4.19.0
     # via napari (setup.cfg)
 jsonschema-specifications==2023.7.1
     # via jsonschema
@@ -317,7 +317,7 @@ pydantic==1.10.12
     #   app-model
     #   napari (setup.cfg)
     #   npe2
-pygments==2.15.1
+pygments==2.16.1
     # via
     #   ipython
     #   napari (setup.cfg)
@@ -410,7 +410,7 @@ qtpy==2.3.1
     #   napari-plugin-manager
     #   qtconsole
     #   superqt
-referencing==0.30.0
+referencing==0.30.2
     # via
     #   jsonschema
     #   jsonschema-specifications
@@ -418,7 +418,7 @@ requests==2.31.0
     # via
     #   pooch
     #   sphinx
-rich==13.5.0
+rich==13.5.2
     # via
     #   napari (setup.cfg)
     #   npe2
@@ -466,7 +466,7 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 stack-data==0.6.2
     # via ipython
-superqt==0.4.1
+superqt==0.5.0
     # via
     #   magicgui
     #   napari (setup.cfg)
@@ -542,7 +542,7 @@ vispy==0.13.0
     #   napari-svg
 wcwidth==0.2.6
     # via prompt-toolkit
-wheel==0.41.0
+wheel==0.41.1
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/constraints/constraints_py3.9.txt b/resources/constraints/constraints_py3.9.txt
index 1bace9f3..b8696e02 100644
--- a/resources/constraints/constraints_py3.9.txt
+++ b/resources/constraints/constraints_py3.9.txt
@@ -43,9 +43,9 @@ click==8.1.6
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.27.0
+cmake==3.27.1
     # via triton
-comm==0.1.3
+comm==0.1.4
     # via ipykernel
 contourpy==1.1.0
     # via matplotlib
@@ -53,7 +53,7 @@ coverage==7.2.7
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.7.1
+dask==2023.8.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -80,7 +80,7 @@ filelock==3.12.2
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.41.1
+fonttools==4.42.0
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -92,7 +92,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.82.0
+hypothesis==6.82.2
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -108,7 +108,7 @@ importlib-metadata==6.8.0
     #   dask
     #   jupyter-client
     #   sphinx
-importlib-resources==6.0.0
+importlib-resources==6.0.1
     # via matplotlib
 in-n-out==0.1.8
     # via app-model
@@ -132,7 +132,7 @@ jinja2==3.1.2
     #   numpydoc
     #   sphinx
     #   torch
-jsonschema==4.18.4
+jsonschema==4.19.0
     # via napari (setup.cfg)
 jsonschema-specifications==2023.7.1
     # via jsonschema
@@ -200,7 +200,7 @@ npe2==0.7.2
     #   napari-plugin-manager
 numcodecs==0.11.0
     # via zarr
-numpy==1.25.1
+numpy==1.25.2
     # via
     #   contourpy
     #   dask
@@ -312,7 +312,7 @@ pydantic==1.10.12
     #   app-model
     #   napari (setup.cfg)
     #   npe2
-pygments==2.15.1
+pygments==2.16.1
     # via
     #   ipython
     #   napari (setup.cfg)
@@ -403,7 +403,7 @@ qtpy==2.3.1
     #   napari-plugin-manager
     #   qtconsole
     #   superqt
-referencing==0.30.0
+referencing==0.30.2
     # via
     #   jsonschema
     #   jsonschema-specifications
@@ -411,7 +411,7 @@ requests==2.31.0
     # via
     #   pooch
     #   sphinx
-rich==13.5.0
+rich==13.5.2
     # via
     #   napari (setup.cfg)
     #   npe2
@@ -459,7 +459,7 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 stack-data==0.6.2
     # via ipython
-superqt==0.4.1
+superqt==0.5.0
     # via
     #   magicgui
     #   napari (setup.cfg)
@@ -535,7 +535,7 @@ vispy==0.13.0
     #   napari-svg
 wcwidth==0.2.6
     # via prompt-toolkit
-wheel==0.41.0
+wheel==0.41.1
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/constraints/constraints_py3.9_examples.txt b/resources/constraints/constraints_py3.9_examples.txt
index 0cd102d5..48ae044e 100644
--- a/resources/constraints/constraints_py3.9_examples.txt
+++ b/resources/constraints/constraints_py3.9_examples.txt
@@ -43,9 +43,9 @@ click==8.1.6
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.27.0
+cmake==3.27.1
     # via triton
-comm==0.1.3
+comm==0.1.4
     # via ipykernel
 contourpy==1.1.0
     # via matplotlib
@@ -53,7 +53,7 @@ coverage==7.2.7
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.7.1
+dask==2023.8.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -80,7 +80,7 @@ filelock==3.12.2
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.41.1
+fonttools==4.42.0
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -92,7 +92,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.82.0
+hypothesis==6.82.2
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -108,7 +108,7 @@ importlib-metadata==6.8.0
     #   dask
     #   jupyter-client
     #   sphinx
-importlib-resources==6.0.0
+importlib-resources==6.0.1
     # via matplotlib
 in-n-out==0.1.8
     # via app-model
@@ -136,7 +136,7 @@ joblib==1.3.1
     # via
     #   nilearn
     #   scikit-learn
-jsonschema==4.18.4
+jsonschema==4.19.0
     # via napari (setup.cfg)
 jsonschema-specifications==2023.7.1
     # via jsonschema
@@ -330,7 +330,7 @@ pydantic==1.10.12
     #   app-model
     #   napari (setup.cfg)
     #   npe2
-pygments==2.15.1
+pygments==2.16.1
     # via
     #   ipython
     #   napari (setup.cfg)
@@ -421,7 +421,7 @@ qtpy==2.3.1
     #   napari-plugin-manager
     #   qtconsole
     #   superqt
-referencing==0.30.0
+referencing==0.30.2
     # via
     #   jsonschema
     #   jsonschema-specifications
@@ -430,7 +430,7 @@ requests==2.31.0
     #   nilearn
     #   pooch
     #   sphinx
-rich==13.5.0
+rich==13.5.2
     # via
     #   napari (setup.cfg)
     #   npe2
@@ -482,7 +482,7 @@ sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
 stack-data==0.6.2
     # via ipython
-superqt==0.4.1
+superqt==0.5.0
     # via
     #   magicgui
     #   napari (setup.cfg)
@@ -560,7 +560,7 @@ vispy==0.13.0
     #   napari-svg
 wcwidth==0.2.6
     # via prompt-toolkit
-wheel==0.41.0
+wheel==0.41.1
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/requirements_mypy.txt b/resources/requirements_mypy.txt
index 94f84de3..255b724b 100644
--- a/resources/requirements_mypy.txt
+++ b/resources/requirements_mypy.txt
@@ -26,7 +26,7 @@ mypy-extensions==1.0.0
     #   psygnal
 npe2==0.7.2
     # via -r resources/requirements_mypy.in
-numpy==1.25.1
+numpy==1.25.2
     # via -r resources/requirements_mypy.in
 packaging==23.1
     # via
@@ -41,7 +41,7 @@ pydantic==1.10.12
     # via
     #   -r resources/requirements_mypy.in
     #   npe2
-pygments==2.15.1
+pygments==2.16.1
     # via
     #   rich
     #   superqt
@@ -54,9 +54,9 @@ qtpy==2.3.1
     #   -r resources/requirements_mypy.in
     #   magicgui
     #   superqt
-rich==13.5.0
+rich==13.5.2
     # via npe2
-superqt==0.4.1
+superqt==0.5.0
     # via magicgui
 tomli-w==1.0.0
     # via npe2
--
2.34.1
